### PR TITLE
Bitbucket REPO_TOKEN update

### PR DESCRIPTION
**Pull request by DavidGOrtega**
After spending time trying to implement OAuth in CML I realised that Bitbucket still supports Basic Auth using [app passwords](https://support.atlassian.com/bitbucket-cloud/docs/app-passwords/) 🤦‍♂️ 

- closes https://github.com/iterative/cml/issues/914

### CODE REVIEW

File: content/docs/self-hosted-runners.md [hunk 1]
```diff
@@ -312,8 +312,8 @@ variable with a Base64 encoded username and password.
 
 Use either:
 
-- your user access credentials (consider using
-  [Bitbucket Cloud App Passwords](https://support.atlassian.com/bitbucket-cloud/docs/app-passwords),
+- your username and a
+  [Bitbucket Cloud App Password](https://bitbucket.org/account/settings/app-passwords/),
```
[the remark this answers is not in the document]
Comment (DavidGOrtega) [reply content/docs/self-hosted-runners.md:316]: Pull requests write
Runners read
Pipelines write
Issues write

But I can do a full battery of test and testify if missing anyone

[the remark this answers is not in the document]
Comment (casperdcl) [reply content/docs/self-hosted-runners.md:316]: Presumably `Repositories Write` is required for `cml ci`? Or is `Pull requests Write` enough?

[the remark this answers is not in the document]
Comment (0x2b3bfa0) [reply content/docs/self-hosted-runners.md:316]: (ping @iterative/cml)

[the remark this answers is not in the document]
Comment (casperdcl) [reply content/docs/self-hosted-runners.md:316]: ?? still waiting on [this](https://github.com/iterative/cml.dev/pull/205#discussion_r828634983)

**Comment by 0x2b3bfa0** [comment]
* https://github.com/iterative/cml/pull/996#issuecomment-1116937806

File: content/docs/self-hosted-runners.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (0x2b3bfa0) [reply content/docs/self-hosted-runners.md:316]: Enough to pass all our tests and account for runner support (https://github.com/iterative/cml/pull/798)
Captura de pantalla 2022-05-04 a las 7 20 46

[the remark this answers is not in the document]
Comment (0x2b3bfa0) [reply content/docs/self-hosted-runners.md:316]: @casperdcl 🏓

[the remark this answers is not in the document]
Comment (0x2b3bfa0) [reply content/docs/self-hosted-runners.md:316]: **Pull requests** — Write
**Pipelines** — Write
**Runners** — Write

(The others are implicitly set)

[the remark this answers is not in the document]
Comment (casperdcl) [reply content/docs/self-hosted-runners.md:316]:
```suggestion
- your username and a
  [Bitbucket Cloud App Password](https://bitbucket.org/account/settings/app-passwords/)
  with `Write` permission for Pull requests, Pipelines, and Runners,
```

**Review by casperdcl — APPROVED** [review]

File: content/docs/self-hosted-runners.md [hunk 2]
```diff
@@ -312,8 +312,9 @@ variable with a Base64 encoded username and password.
 
 Use either:
 
-- your user access credentials (consider using
-  [Bitbucket Cloud App Passwords](https://support.atlassian.com/bitbucket-cloud/docs/app-passwords),
+- your username and a
+  [Bitbucket Cloud App Password](https://bitbucket.org/account/settings/app-passwords/)
+  with `Write` permission for Pull requests, Pipelines, and Runners,
   or
```
Comment (casperdcl) [inline content/docs/self-hosted-runners.md:318]:
```suggestion
  with `Write` permission for Pull requests, Pipelines, and Runners, or
```

File: content/docs/self-hosted-runners.md [hunk 3]
```diff
@@ -312,9 +312,9 @@ variable with a Base64 encoded username and password.
 
 Use either:
 
-- your user access credentials (consider using
-  [Bitbucket Cloud App Passwords](https://support.atlassian.com/bitbucket-cloud/docs/app-passwords),
-  or
+- your username and a
+  [Bitbucket Cloud App Password](https://bitbucket.org/account/settings/app-passwords/)
+  with `Write` permission for Pull requests, Pipelines, and Runners, or
```
[the remark this answers is not in the document]
Comment (casperdcl) [reply content/docs/self-hosted-runners.md]: nooo! Oxford commas are part of the spec![^1]

[^1]: https://github.com/iterative/dvc.org/pull/3484

File: content/docs/self-hosted-runners.md [hunk 4]
```diff
@@ -312,9 +312,9 @@ variable with a Base64 encoded username and password.
 
 Use either:
 
-- your user access credentials (consider using
-  [Bitbucket Cloud App Passwords](https://support.atlassian.com/bitbucket-cloud/docs/app-passwords),
-  or
+- your username and a
+  [Bitbucket Cloud App Password](https://bitbucket.org/account/settings/app-passwords/)
+  with `Write` permission for Pull requests, Pipelines and Runners, or
```
Comment (0x2b3bfa0) [inline content/docs/self-hosted-runners.md:317]:
```suggestion
  with `Write` permission for Pull requests, Pipelines, and Runners, or
```

**Pull request merged**